game_eggs/steamcmd_servers/the_isle/evrima/egg-the-isle.json

```diff
@@ -15,14 +15,14 @@
     "file_denylist": [],
     "startup": "\/home\/container\/steamcmd\/steamcmd.sh +force_install_dir \/home\/container +login Anonymous +app_update 412680 -beta evrima +quit && \/home\/container\/TheIsle\/Binaries\/Linux\/TheIsleServer-Linux-Shipping -Port=$SERVER_PORT -ini:Engine:[EpicOnlineServices]:DedicatedServerClientId=xyza7891gk5PRo3J7G9puCJGFJjmEguW -ini:Engine:[EpicOnlineServices]:DedicatedServerClientSecret=pKWl6t5i9NJK8gTpVlAxzENZ65P8hYzodV8Dqe5Rlc8",
```

Comment:
remove `/home/container/steamcmd/steamcmd.sh +force_install_dir /home/container +login Anonymous +app_update 412680 -beta evrima +quit &&` from the startup. the image will do this if the needed variables ( app id, auto update , and beta) are set correctly